### DISCUSSION TITLE
How to add/remove boolean attribute to an element?

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#835] [Q&A] [answered]
Hi

I saw how to set the value of an attribute with **mv-attr-*** [here](https://codepen.io/dmitrysharabin/pen/gOoqOzj?editors=1000):

But in case of Boolean attribute , like :
1/ "**checked**" inside:    `<input type="checkbox" checked> `
2/ "**open**" inside: ` <details open>`

is there a way to add/remove the attribute itself?

Example 1:
in case of:
`<details></details>`

result should be:
`<details open></details>`

Thanks

PS: setting `open="false"`, doesn't work: I really need to remove '`open`' itself

**Comment by DmitrySharabin** [comment, marked as answer]
Hey there,

have you tried `<input type="checkbox" checked="[true]" />`? Actually, any expression that resolves to boolean will do. So, you don’t need `mv-attr-*` for this (by the way, it’s a work in progress and is not at the core yet).

**Reply by GalinhaLX** [reply]
The browser doesn't care what value assigned to checked attribute, as soon as it sees checked in the checkbox input tag, it's flagged as checked.

**Reply by DmitrySharabin** [reply]
True. However, if the value of this attribute is an expression, Mavo will add/remove it for you automatically. Just play with it.

**Reply by GalinhaLX** [reply]
Interresting to know that, thanks
Usefull for <details open></details>

**Comment by GalinhaLX** [comment]
For information: I'm trying to build a "accordion" section which is editable in both summary/details sections in edition Mode of mavo
But in read mode, only first detail should be expended

mavo site is using it here:
https://github.com/mavoweb/mavo.io/blob/32ac8d6f3aabce2bea9a919a34e382b1b07faf20/docs/functions/index.html#L48

but this code, is too complicated for my level :(

**Reply by DmitrySharabin** [reply]
This might help: https://codepen.io/mavoweb/pen/MXbKKO

**Reply by GalinhaLX** [reply]
Great 

many thanks @DmitrySharabin
